lib/media/segment_reference.js

```diff
@@ -429,6 +432,25 @@ shaka.media.SegmentReference = class {
     return this.independent;
   }
 
+  /**
+   * Mark the reference as byterange optimization.
```

Comment:
This could explain that, the "byterange optimization" means that it is playable using MP4 low latency streaming with chunked data.
The extra explanation here would be helpful since this JSDoc will end up inside our documentation, as this is an exported method on an exported class. Meanwhile, the link to that Akamai blog entry is buried inside the code for the HLS parser, and thus much less visible to developers.

Reply:
Done!